run/events-pubsub/README.md

```diff
@@ -0,0 +1,35 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes Pub/Sub messages.
+
+For more details on how to work with this sample read the [Google Cloud Run Node.js Samples README](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/run).
+
+## Dependencies
+
+* **express**: Web server framework.
+* **body-parser**: express middleware for request payload processing.
+* **mocha**: [development] Test running framework.
+* **supertest**: [development] HTTP assertion test client.
+
+## Quickstart
+
+```sh
```

Comment:
Instructions or comments on the steps would be nice

Reply:
Good point. Added some instructions.